src/controllers/channel.controller.js

```diff
@@ -244,9 +244,12 @@ exports.removeUserFromChannel = async (req, res) => {
 
 exports.getUsers = async (req, res) => {
   try {
-    const ChannelId = req.channel._id;
+    //console.log(req.body);
+    const ChannelId = req.body.channel;
```

Comment:
req.body.channel -> req.body.channel._id    I think this will be channel id.

Reply:
But I tested it and it was running